dockerfile-generator/dockerfile-gen-services/src/main/java/io/hyscale/dockerfile/gen/services/manager/impl/ArtifactManagerImpl.java

```diff
@@ -78,6 +80,24 @@
         return supportingFiles;
     }
 
+    private void validate(List<Artifact> artifactsList) throws HyscaleException {
+        if (artifactsList == null || artifactsList.isEmpty()) {
+            return;
+        }
+        // Artifacts should exist
+        List<String> artifactsNotFound = new ArrayList<String>();
```

Comment:
This was a bug fix too ! Please raise an issue for this @tushar-tushar

Reply:
Created a new issue: https://github.com/hyscale/hyscale/issues/231